### PR TITLE
Updated commands for VSM section.

### LINKED ISSUE #187: Update README with all commands
amongst others the `--script-readable` is missing from `vagrant service-manager env docker`

`box` and `status` are missing.

@Preeticp please review the help for this.  

```
bexelbie@bexelbie:~/Vagrant/adb-1.8rc$ vagrant service-manager -h
Usage: vagrant service-manager <command> [options]

Commands:
     env          displays connection information for services in the box
     box          displays version and release information for the box
     restart      restarts the given systemd service in the box
     status       list services and their running state

Options:
     -h, --help   print this help

For help on any individual command run `vagrant service-manager COMMAND -h`
bexelbie@bexelbie:~/Vagrant/adb-1.8rc$ vagrant service-manager box -h
Usage: vagrant service-manager box <sub-command> [options]

Sub-Command:
      version    display version and release information about the running VM
      ip         display routable IP address of the running VM

Options:
      --script-readable  display information in a script readable format
      -h, --help         print this help
bexelbie@bexelbie:~/Vagrant/adb-1.8rc$ vagrant service-manager status -
- - stopped
bexelbie@bexelbie:~/Vagrant/adb-1.8rc$ vagrant service-manager status -h
Usage: vagrant service-manager status [service] [options]

Options:
      -h, --help         print this help

If a service is provided, only that service is reported.
If no service is provided only supported orchestrators are reported.

Examples:
      vagrant service-manager box version
      vagrant service-manager box ip
      vagrant service-manager box version --script-readable
bexelbie@bexelbie:~/Vagrant/adb-1.8rc$ vagrant service-manager restart -h
Restarts the service

Usage: vagrant service-manager restart <object> [options]

Object:
    Object is a service provided via sccli or systemd. For example:
     docker, k8s, or openshift etc.

Options:
      -h, --help         print this help

Examples:
  vagrant service-manager restart docker
```

### PR DESCRIPTION
**Pull request by Preeticp**
Fixes #187

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,14 +55,19 @@ developer's workstation without having to `ssh` directly into the ADB virtual ma
...
------------------------------------------------------------|-----------------------------------------
-`vagrant service-manager env`                              | Displays connection information for all active providers in the box.
-`vagrant service-manager env docker`                       | Displays connection information for the Docker provider.
-`vagrant service-manager env openshift` [--script-readable]| Displays connection information for the OpenShift provider. This is optionally available in script readable format too.
-`vagrant service-manager box version` [--script-readable]  | Displays the version and release of the running Vagrant box. This is optionally available in script readable format too.
+The following section lists the available commands for the plugin and their explanation:
+
+1. `vagrant service-manager env [service] [--script-readable]` - Displays connection information for all active services in the box in a manner that can be evaluated in a shell. If a service is specified only the information for that service is displayed. When `--script-readable` is specified the output is in key=value format. Supported services are: Docker; OpenShift
```
Comment (brgnepal) [inline README.md]: - Probably if you put "service" and "key=value" in quote here that would be more readable.

```
.... If a service(should as `service`) is specified only the information for .. ......... is specified the output is in key=value(`key=value`) format...
```
- I think Supported services should be in comma and this line show be in new line.

**Comment by navidshaikh** [comment]
Add an entry in CHANGELOG as well @Preeticp

File: README.md [hunk 2]
```diff
@@ -55,14 +55,19 @@ developer's workstation without having to `ssh` directly into the ADB virtual ma
...
+
+2. `vagrant service-manager box [command]` - Displays box related information like release version, ip etc.
```
Comment (brgnepal) [inline README.md]: Bex corrected me as :

```
Displays box related information like version, release, IP etc.
```

Refer https://github.com/projectatomic/vagrant-service-manager/pull/193/files

File: README.md [hunk 3]
```diff
@@ -55,14 +55,19 @@ developer's workstation without having to `ssh` directly into the ADB virtual ma
...
+The following section lists the available commands for the plugin and their explanation:
+
+1. `vagrant service-manager env [service] [--script-readable]` - Displays connection information for all active services in the box in a manner that can be evaluated in a shell. If a service is specified only the information for that service is displayed. When `--script-readable` is specified the output is in key=value format. Supported services are: Docker; OpenShift
+
+2. `vagrant service-manager box [command]` - Displays box related information like release version, ip etc. 
+
+3. `vagrant service-manager box version [--script-readable]` - Displays the version and release information of the running VM. When `--script-readable` is specified the output is in key=value format.
+
+4. `vagrant service-manager box ip [--script-readable]` - Displays the routable IP address of the running VM. When `--script-readable` is specified the output is in key=value format.
+
```
Comment (navidshaikh) [inline README.md:66]: `box ip` subcommand does not support `--script-readable` option. 

Please rewrite as
4. `vagrant service-manager box ip` - Displays the routable IP address of the running VM.

File: README.md [hunk 4]
```diff
@@ -55,14 +55,19 @@ developer's workstation without having to `ssh` directly into the ADB virtual ma
...
-`vagrant service-manager env openshift` [--script-readable]| Displays connection information for the OpenShift provider. This is optionally available in script readable format too.
-`vagrant service-manager box version` [--script-readable]  | Displays the version and release of the running Vagrant box. This is optionally available in script readable format too.
+The following section lists the available commands for the plugin and their explanation:
+
+1. `vagrant service-manager env [service] [--script-readable]` - Displays connection information for all active services in the box in a manner that can be evaluated in a shell. If a service is specified only the information for that service is displayed. When `--script-readable` is specified the output is in key=value format. Supported services are: Docker; OpenShift
+
+2. `vagrant service-manager box [command]` - Displays box related information like release version, ip etc. 
+
+3. `vagrant service-manager box version [--script-readable]` - Displays the version and release information of the running VM. When `--script-readable` is specified the output is in key=value format.
```
Comment (brgnepal) [inline README.md]: "key=value" to

```
`key=value`
```

Also in step 4.

Comment (navidshaikh) [inline README.md]: Yes please.

File: README.md [hunk 5]
```diff
@@ -55,14 +55,19 @@ developer's workstation without having to `ssh` directly into the ADB virtual ma
...
+
+4. `vagrant service-manager box ip [--script-readable]` - Displays the routable IP address of the running VM. When `--script-readable` is specified the output is in key=value format.
+
+5. `vagrant service-manager status [service]` - Lists services and their running state. If a service is specified only the status of that service is displayed. If no service is provided only supported orchestrators are reported.
```
Comment (brgnepal) [inline README.md]: `If no service is provided only supported orchestrators are reported.` to 

```
If no service is provided then only supported orchestrators are reported.
```

**Comment by Preeticp** [comment]
@budhrg, I have updated the commands section with the suggested changes. Also have changed the format of the section for better readability.
Thanks for your input.

File: CHANGELOG.md [hunk 6]
```diff
@@ -10,6 +10,7 @@
 - Fix #135: Refactor command.rb to make commands easier to add/maintain @budhrg
 - Adds @budhrg as co-maintainer for the plugin @navidshaikh
 - Fix #191: 'vagrant service-manager restart' not handled correctly @budhrg
+- Fixes # 187, Updated commands in the Available Commands section.
```
Comment (brgnepal) [inline CHANGELOG.md]: This should be 

```
Fixes #187: Updated commands in the Available Commands section @Preeticp
```

File: README.md [hunk 7]
```diff
@@ -55,14 +54,30 @@ developer's workstation without having to `ssh` directly into the ADB virtual ma
...
+Displays box related information like release version, IP etc. 
+
+3. `vagrant service-manager box version [--script-readable]` 
+Displays the version and release information of the running VM. When `--script-readable` is specified the output is in `key=value` format.
+
+4. `vagrant service-manager box ip` 
+Displays the routable IP address of the running VM. 
+
+5. `vagrant service-manager status [service]` 
+Lists services and their running state. If a `[service]` is specified only the status of that service is displayed. If no service is provided then only supported orchestrators are reported.
```
Comment (brgnepal) [inline README.md]: No need of brackets in service (`[service]`). It should be just service enclosed in back tick.

File: README.md [hunk 8]
```diff
@@ -55,14 +54,30 @@ developer's workstation without having to `ssh` directly into the ADB virtual ma
...
+
+6. `vagrant service-manager restart [service]` 
+Restarts the given service in the box.
```
Comment (budhrg) [inline README.md]: Are you trying to show "Restarts the given service in the box." in new line?
If yes, then You need to add empty line between "6.  `vagrant service-manager restart [service]`" and "Restarts the given service in the box.". As, right now it is coming in one line check https://github.com/Preeticp/vagrant-service-manager/blob/vsm-docs-reorg/README.md#available-commands.

File: README.md [hunk 9]
```diff
@@ -55,14 +54,30 @@ developer's workstation without having to `ssh` directly into the ADB virtual ma
 
 ## Available Commands <a name="commands"></a>
 
-The following table lists the available commands for the plugin and their explanation:
+The following section lists the available commands for the plugin and their explanation:
```
Comment (brgnepal) [inline README.md:57]: Probably you would like to update this section (Available commands) like this one https://gist.github.com/budhrg/1321e995492a86c0df289376e9666531#available-commands.

If you want, just edit this "gist", and add the changes. I have updated this gist with required changes for unnecessary brackets

File: CHANGELOG.md [hunk 10]
```diff
@@ -10,6 +10,7 @@
 - Fix #135: Refactor command.rb to make commands easier to add/maintain @budhrg
 - Adds @budhrg as co-maintainer for the plugin @navidshaikh
 - Fix #191: 'vagrant service-manager restart' not handled correctly @budhrg
+- Fixes #187, Updated commands in the Available Commands section.
```
Comment (brgnepal) [inline CHANGELOG.md]: You forgot your user name here.

**Comment by Preeticp** [comment]
Made the changes @budhrg

**Comment by brgnepal** [comment]
LGTM :+1:

**Comment by navidshaikh** [comment]
LGTM

**Pull request merged**